Content.Server/DeltaV/Objectives/Systems/NukeStationConditionSystem.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Content.Server.DeltaV.Objectives.Components;
+using Content.Shared.Objectives.Components;
+
+namespace Content.Server.DeltaV.Objectives.Systems;
+
+public sealed class NukeStationConditionSystem : EntitySystem
+{
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        SubscribeLocalEvent<NukeStationConditionComponent, ObjectiveGetProgressEvent>(OnGetProgress);
+    }
+
+    // TODO: Make this work!
+    private void OnGetProgress(Entity<NukeStationConditionComponent> condition, ref ObjectiveGetProgressEvent args)
```

Comment:
add CodeConditionComponent to the objective then when nuke explodes check if the map its on is correct and then greentext all nuke objectives (should probably store the station on the objective to check it on nuke event)